common/dynamicconfig/constants.go

```diff
@@ -825,15 +810,21 @@ of Timeout and if no activity is seen even after that the connection is closed.`
 		false,
 		`FrontendEnableNexusAPIs enables serving Nexus HTTP requests in the frontend.`,
 	)
-	FrontendRefreshNexusIncomingServicesLongPollTimeout = NewGlobalDurationSetting(
-		"frontend.refreshNexusIncomingServicesLongPollTimeout",
+	EnableNexusEndpointRegistryBackgroundRefresh = NewGlobalBoolSetting(
```

Comment:
Why have a separate config for this? Would we ever want it to be out of sync with `FrontendEnableNexusAPIs`?

Reply:
History service needs this config too, I don't want to use a frontend config for it.

That being said, I want to try to figure out a way to avoid needing this config entirely.